tests/data/src/pep518-3.0/setup.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
 
```

Comment:
There's a lot of churn like this in the PR. I'm a strong -1 on churn like this when it's purely cosmetic (I'm sure there's some sort of style guide recommendation that names should be sorted, or some such) but it's not one we've applied historically, and there's nothing in this PR that specifically talks about applying new style rules.

If you want to apply new style rules, can the change be split out into a separate PR and debated as a deliberate change, not added as part of an unrelated change? (I assume this was the fault of the flake8 update changing the default rules? We shouldn't blindly accept changes to the defaults like this - we should probably have an explicit config that defines our rules - but as I say, that's a debate for a PR specific to style rule changes).